packages/editor/src/browser/editor-navigation-contribution.ts

```diff
@@ -96,6 +106,24 @@ export class EditorNavigationContribution implements Disposable, FrontendApplica
         this.toDispose.dispose();
     }
 
+    /**
+     * Toggle the editor word wrap behavior.
+     */
+    protected async toggleWordWrap(): Promise<void> {
+        // Get the current word wrap.
+        const wordWrap: string | undefined = this.preferenceService.get('editor.wordWrap');
+        if (typeof wordWrap === 'undefined') {
```

Comment:
i am ok with this but why do we not simply do `wordWrap === undefined`

Reply:
if someone redefine what `undefined` like `window.undefined = "foo";` 😄 

We don't expect it in Theia, you can just test with `wordWrap === undefined`

Reply:
@marechal-p was convincing me that it might be better practice or safer to use `typeof` guards for exactly the reason you mentioned Anton :smile:

Reply:
We discussed it 3 year ago and agreed not doing `window.undefined = "foo"`. Right now it does not make sense to use `typeof` already, our code base full of `wordWrap === undefined` check, adding it in one place does not make it safer. Also for 3 years such issue never did not arise.

Reply:
> We discussed it 3 year ago and agreed not doing `window.undefined = "foo"`. Right now it does not make sense to use `typeof` already, our code base full of `wordWrap === undefined` check, adding it in one place does not make it safer. Also for 3 years such issue never did not arise.

Sounds fine, I'll make the small update.